concepticondata/conceptlists/Miklashevsky-2018-506.tsv

```diff
@@ -0,0 +1,507 @@
+Miklashevsky-2018-506-178	178	кукуруза	kukuruza	corn	Food	concrete	8	f	0.9	3.97	1.38	4.66	6.12	4.03	6.79	4.73	2.94	5.2	2.97	506	MAIZE
+Miklashevsky-2018-506-179	179	кукушка	kukushka	cuckoo	Animal	concrete	7	f	0.7	4.07	4.61	1.69	1.29	2.37	6	2.73	4.84	4.42	3.2	2934	CUCKOO
+Miklashevsky-2018-506-180	180	курага	kuraga	dried apricots	Food	concrete	6	f	0.3	4.38	1.1	4.62	5.88	3.69	6.79	4.47	2.73	6.07	2.66		
+Miklashevsky-2018-506-181	181	курица	kuritsa	hen	Animal	concrete	6	f	1.5	4.6	3.11	4.7	4.93	3.55	6.62	3.32	2.38	3.93	4.96	1514	HEN
+Miklashevsky-2018-506-182	182	лампочка	lampochka	bulb	Tool_power_grip	concrete	8	f	1.4	4.93	1.9	1.93	1.62	4.83	6.96	5.31	4.81	4	5.25	2617	BULB
+Miklashevsky-2018-506-183	183	ластик	lastik	eraser	Tool_precise_grip	concrete	6	m	0.3	4.27	2.57	2.35	1.96	4.74	6.48	6.68	2.84	5.38	3.9		
+Miklashevsky-2018-506-184	184	лев	lev	lion	Animal	concrete	3	m	1.8	4.38	4.63	3.17	1.72	3.25	6.8	1.77	2.97	3.63	4.43	1386	LION
+Miklashevsky-2018-506-185	185	ледокол	ledokol	ice-breaker	Transport	concrete	7	m	0.6	3.52	3.6	1.66	1.55	2.58	5.97	4.6	2.73	7.34	2.63		
+Miklashevsky-2018-506-186	186	лемур	lemur	lemur	Animal	concrete	5	m	0	5.17	3	2.31	1.21	2.63	5.9	2.73	4.44	7.15	2.17		
+Miklashevsky-2018-506-187	187	леопард	leopard	leopard	Animal	concrete	7	m	0.5	4.8	3.37	1.9	1.6	3.03	6.34	1.94	2.84	5.7	2.55	1139	LEOPARD
+Miklashevsky-2018-506-188	188	лимон	limon	lemon	Food	concrete	5	m	1.1	4.57	1.46	5.65	6.43	3.81	6.9	5.08	3.59	3.58	5.67	3091	LEMON
+Miklashevsky-2018-506-189	189	линейка	lineyka	ruler	Tool_precise_grip	concrete	7	f	1	4.1	1.86	1.34	1.58	4.41	6.62	6.2	2.64	4.83	4.21	3405	RULER
+Miklashevsky-2018-506-190	190	лиса	lisa	fox	Animal	concrete	4	f	1	4.53	2.37	1.93	1.73	3.28	6.66	2.65	2.56	3.85	2.66	1312	FOX
+Miklashevsky-2018-506-191	191	лифт	lift	elevator	Transport	concrete	4	m	1.5	3.79	3.57	2.03	1.69	3.42	6.53	5.57	4.45	5.1	5.83		
+Miklashevsky-2018-506-192	192	лицо	litso	face	Body_Part	concrete	4	n	2.9	5.37	1.82	2.23	1.71	5.22	6.46	4.22	4.13	2.96	6.4	1560	FACE
+Miklashevsky-2018-506-193	193	лоб	lob	forehead	Body_Part	concrete	3	m	1.9	4.07	1.79	1.35	1.57	4.37	6.72	4.03	4.34	3.27	5.63	123	FOREHEAD
+Miklashevsky-2018-506-194	194	лодка	lodka	boat	Transport	concrete	5	f	1.8	4.27	2.85	2.2	1.4	3.55	6.38	5.74	2.28	4.59	2.9	1844	BOAT
+Miklashevsky-2018-506-195	195	ложка	lozhka	spoon	Tool_precise_grip	concrete	5	f	1.6	4.28	3	1.76	3.72	5.63	6.8	6.8	2.3	2.8	6.47	1378	SPOON
+Miklashevsky-2018-506-196	196	локомотив	lokomotiv	locomotive	Transport	concrete	9	m	1.2	4.26	4.71	2.13	1.37	2.54	5.72	4.4	3.33	8.4	2.38		
+Miklashevsky-2018-506-197	197	лопата	lopata	shovel	Tool_power_grip	concrete	6	f	1.3	3.83	2.03	1.76	1.67	4.86	6.48	6.5	2.58	4.73	2.97	1901	SHOVEL
+Miklashevsky-2018-506-198	198	лось	los	elk	Animal	concrete	4	m	0.9	4.67	3.1	2.79	3.5	3.21	6.11	2.53	3.14	5.03	2.89	1772	ELK
+Miklashevsky-2018-506-199	199	лошадь	loshad	mare	Animal	concrete	6	f	1.9	4.38	4.3	4.62	2.62	4.5	6.77	3.57	2.67	4	4.4	938	MARE
```

Comment:
Interesting, I based the mapping on the link between _лось_ to ELK in Dellert-2017-1016-143. I'll unmap it here. Dellert also has a German gloss (_Elch_), so I'll leave this one as is.

Reply:
The ELK mapping is completely fine, but I was referring to the horse-glosses. Dellert has just one such a concept there, which is a general HORSE, no sex distinctions in regard to this animal.

Reply:
Ah ok, thanks for clarifying. Luniewska-2016-299-72 also does not make the female/male distinction. I map ELK again.

Reply:
Sorry for this confusion. I still think that _конь_ in Miklashevsky-2018-506-166 should rather be unmapped as it's the male counterpart of MARE but more general than STALLION. I checked the other lists with this gloss and it is usually the only one referring to the horse as an animal and mapped for this reason very generally to HORSE.

Reply:
Ok, I'll unmap it.

Reply:
@Kristina-Pianykh, in my understanding, "лошадь" is female, but also what one generally uses, right? I mean, a knight travels on the лошадe, etc. So We would have a case of a double-concept, such as "horse or mare". Similar to German "Katze" which is also generic, but also a female version, or "Hund", which is the same with the male prototypic idea of a dog.

Reply:
I think one can defend ONE way of acting here. If for example all animals appear with their counterparts, we can argue that they have been selected based on their primary meaning of opposition by sex (I did not check what they do with "dog", but this is also tricky in Russian, since there's a female and a male term, and it is not clear which is used more frequently, so one could even map both to generic dog). If such a strategy is followed, I'd mention this in the note. As an alternative general rule, I recommend to think of the **most frequent** use-case of a given word, the main concept where the word usually refers to.

Reply:
Yes, I thought about the Katze case here but in this list we have two horse concepts: horse-male (i.e. _конь_) and hourse-female (i.e. _лошадь_). It is the presence of the male horse that disambiguates the other horse concept, which in this specific case would refer then to a female horse.

Reply:
In other words, it would be the same to map Katze to the cat-female if there was Kater in the same list.

Reply:
So rather than unmapping kon', I'd propose to map it to stallion. This would correct the translation of the authors, but be more consistent, as I find it also strange that we end up with having MARE in the list but not the STALLION?

Reply:
Yes, we'd map Katze to "female cat" and "Kater" to "male cat". In the same way, as I'd say it is better to map "kon'" to "stallion" now, although it is not exactly the same as "hengst" in German (which is rarely used).

Reply:
The reason against mapping it to STALLION is that STALLION is too specific for the word _конь_ as it refers to a castrated male horse while _конь_ is a male horse in a very general sense. But with the most animal concepts in this list (including the dog), no such sex distinctions are made. We have a goat concept, though, that is currently mapped to FEMALE GOAT but since there is no other (male) goat instance, it would make sense to remap it to just GOAT, right?

Reply:
Wait, I thought Stallion was like German Hengst, so a male horse. If this is NOT the case, we need to add the concept MALE HORSE to concepticon. Ah, just checked it: we define STALLION as "An **un**castrated male horse." But if it turns out that the aspect of castration is too important, I'd argue that we re-think our concept mapping here and add a new concept for male horse as a general term.

Reply:
And yes: if the goat is mapped to female goat, although this is the prototype (as it is in Russian) it should be remapped to generic goat.

But if we say: unlink "kon'", I'd argue: we should map "loshad'" to HORSE, as I think it mostly translates to horse and not to mare in common usage.

Reply:
In German, we have Wallach (male castrated horse), Stute (female horse), Pferd (horse), and Hengst (uncastrated male horse / stallion). The castration is important because stallions behave very differently from castrated male horses (i.e., stallions can't be integrated into a herd).

Reply:
All mappings in Concepticon seem to refer specifically to [STALLION](https://concepticon.clld.org/parameters/936).

Reply:
Then there are two ways to go about this issue: either to leave "kon'" unmapped and map "loshad" to HORSE or map "kon'" to a new concept set that would be called roughly MALE HORSE (not distinguishing between castrated and uncastrated horses). But maybe we could look into this in a separate issue if we have to review the concepts already mapped to STALLION?

Reply:
I added a new issue and mapped лошадь to HORSE and коза to GOAT.